templates/taxi/manufacturer_list.html

```diff
@@ -1,33 +1,41 @@
 {% extends "base.html" %}
 
 {% block content %}
-    <h1>Manufacturer List
-    </h1>
+  <h1>Manufacturer List
+    <a href="{% url 'taxi:manufacturer-create' %}">+</a>
+  </h1>
```

Comment:
Code Style: It would be better to add a little bit more space between the '+' button and the heading. You can achieve this by adding some CSS styling.